spec/ibandit/swedish_details_converter_spec.rb

```diff
@@ -21,6 +28,16 @@
         its([:swift_account_number]) { is_expected.to eq('00000012810105723') }
       end
 
+      context 'with a separate branch code' do
+        let(:branch_code) { '1281' }
+        let(:account_number) { '0105723' }
+
+        its([:account_number]) { is_expected.to eq('105723') }
```

Comment:
Hmm, @greysteil sourced the data for that `swedish_bank_lookup` file - was that intended to affect local details too?